doc/gtdb.md

```diff
@@ -0,0 +1,137 @@
+
+[**GTDB**](https://gtdb.ecogenomic.org/) (Genome Taxonomy Database) ([Parks et al., 2018](https://www.nature.com/articles/nbt.4229)) is a standardized taxonomy system for bacteria and archaea. It is based on the phylogenetic trees of genomes, hence better reflecting the evolutionary relationships among microorganisms, compared to conventional taxonomy. Our research shows that GTDB shares high consistency with the [WoL](https://biocore.github.io/wol/) phylogeny (which was built using more marker genes and more expensive methods) ([Zhu et al., 2019](https://www.nature.com/articles/s41467-019-13443-4)).
+
+The current release of GTDB is [**R04-RS89**](https://data.ace.uq.edu.au/public/gtdb/data/releases/release89/) (version 4, indexed to RefSeq release 89), when this tutorial is written, and on which the tutorial is based. This release contains 143,512 bacterial and 2,392 archaeal organisms in its taxonomy, or 23,458 bacterial and 1,248 archaeal genomes in its collection of actual genome sequences. These 24,706 "**species clusters**" cover the complete GTDB taxonomic framework ([Parks et al., 2020](https://www.nature.com/articles/s41587-020-0501-8)).
+
+This tutorial will explain various strategies of utilizing GTDB taxonomy / phylogeny in a meta'omics analysis with Woltka.
+
+
+## Contents
+
+- [Woltka directly parses GTDB taxonomy](#woltka-directly-parses-gtdb-taxonomy)
+- [Reformat GTDB as taxdump style](#reformat-gtdb-as-taxdump-style)
+- [Build GTDB genome database](#build-gtdb-genome-database)
+- [Use GTDB phylogeny](#use-gtdb-phylogeny)
+- [Use WoL database with GTDB taxonomy](#use-wol-database-with-gtdb-taxonomy)
+
+
+## Woltka directly parses GTDB taxonomy
+
+The GTDB taxonomy ([`bac120_taxonomy_r89.tsv`](https://data.ace.uq.edu.au/public/gtdb/data/releases/release89/89.0/bac120_taxonomy_r89.tsv) and [`ar122_taxonomy_r89.tsv`](https://data.ace.uq.edu.au/public/gtdb/data/releases/release89/89.0/ar122_taxonomy_r89.tsv)) is provided in the format of **Greengenes-style lineage strings**. For example: _Escherichia coli_ O157:H7 str. Sakai (NCBI RefSeq assembly: [GCF_000008865.1](https://www.ncbi.nlm.nih.gov/assembly/GCF_000008865.1/)) is classified as:
+
+```
+RS_GCF_000008865.1	d__Bacteria;p__Proteobacteria;c__Gammaproteobacteria;o__Enterobacterales;f__Enterobacteriaceae;g__Escherichia;s__Escherichia dysenteriae
+```
+
+This format can be directly parsed by Woltka:
+
+```bash
+woltka classify --lineage bac120_taxonomy_r89.tsv ...
+```
+
+If you work with both Bacteria and Archaea (this is the common case), you may simply concatenate the two taxonomy files before use:
+
+```bash
+cat bac120_taxonomy_r89.tsv ar122_taxonomy_r89.tsv > gtdb_taxonomy.tsv
+```
+
+Note: The [GTDB-Tk](https://github.com/Ecogenomics/GtdbTk) database ([`gtdbtk_data.tar.gz`](https://data.ace.uq.edu.au/public/gtdb/data/releases/release89/89.0/gtdbtk_r89_data.tar.gz)) already provides a combined `gtdb_taxonomy.tsv` file.
+
+
+## Reformat GTDB as taxdump style
```

Comment:
Is it a style or a format?

Reply:
It is a format, but it isn't a format that's widely adopted. Instead, to my knowledge, only NCBI uses it. And given the popularity of NCBI, I tended to mention it as a "style".